README.md

```diff
@@ -1,131 +1,206 @@
+Currently, this project is in-progress and not feature complete. There is still
+some work to do before it is ready for a general audience.
 
 ## Dependencies
+
 ### Run Time
-* GTKmm
-* JsonCpp
-* libapparmor
-* [libappanvil](https://github.com/jack-ullery/libappanvil)
-* apparmor-utils
+
+- GTKmm
+- JsonCpp
+- libapparmor
+- [libappanvil](https://github.com/jack-ullery/libappanvil)
+- apparmor-utils
 
 #### Install commands (Ubuntu)
+
 ```
 sudo apt install libgtkmm-3.0-dev libjsoncpp-dev libapparmor-dev apparmor-utils
 ```
 
 #### Install commands (OpenSUSE)
+
 ```
 sudo zypper in gtkmm3-devel jsoncpp-devel libapparmor-devel apparmor-utils
 ```
 
 ### Compile Time
-* CMake
-* PkgConfig
-* clang (or another equivalent c++ compiler)
+
+- CMake
+- PkgConfig
+- clang (or another equivalent c++ compiler)
 
 #### Install commands (Ubuntu)
+
 ```
 sudo apt install pkg-config cmake clang
 ```
 
 #### Install commands (OpenSUSE)
+
 ```
 sudo zypper in cmake clang
 ```
 
 ### Optional
+
 A GUI builder
-* glade 
+
+- glade
 
 Linters and Static Analysis tools used by the Makefile
-* cppcheck
-* clang-tidy
+
+- cppcheck
+- clang-tidy
 
 Code beautifier/formatter
-* clang-format
+
+- clang-format
 
 Unit Tests
-* googletest
-* googlemock
+
+- googletest
+- googlemock
 
 Code Coverage
-* lcov
+
+- lcov
 
 #### Install commands (Ubuntu)
+
 ```
 sudo apt install glade cppcheck clang-tidy clang-format libgtest-dev libgmock-dev lcov
 ```
 
 #### Install commands (OpenSUSE)
+
 ```
 sudo zypper in glade cppcheck clang gtest gmock lcov
 ```
 
 # Compilation Instructions
+
 ### Prebuild
-Before you build the executable, you must first generate the makefile by running:
+
+Before you build the executable, you must first generate the makefile by
+running:
+
 ```
 cmake .
 ```
+
 Alternatively, if you want to run optional linters and static analysis checks:
+
 ```
 cmake -DANALYZE=TRUE .
 ```
 
 ### Build
+
 After the makefile is generated, you can build and install the executables:
+
 ```
 sudo make install
 ```
-This builds the necessary executables, installs them, and applies recommended changes to PolicyKit.
+
+This builds the necessary executables, installs them, and applies recommended
+changes to PolicyKit.
 
 ### Test
+
 To build the unit tests:
+
 ```
 make test
 ```
+
 To generate a code coverage report:
+
 ```
 make report
 ```
-Afterwards, you can then view the report by opening `./report/index.html` inside a web browser.
+
+Afterwards, you can then view the report by opening `./report/index.html` inside
+a web browser.
 
 ### Automatically Format Code
+
 To format code, without building it:
+
 ```
 make FORMAT
 ```
+
+## Packages
+
+Below are the packaging details for the AppAnvil project across different
+platforms.
+
+### Archlinux (AUR)
+
+To install AppAnvil on Arch Linux, you can use the package available in the AUR.
+
+#### Using Pamac AUR Helper
+
+```bash
+sudo pamac install appanvil-git
+```
```

Comment:
Please correct me if I'm wrong, but I thought pamac mostly by Manjaro (which is Arch-based).

Also, I know Arch-wiki warns users to avoid AUR helpers in [this page](https://wiki.archlinux.org/title/AUR_helpers).

Users would also need to manually enable AUR packages for pamac to find this.